crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/panic_data.rs

```diff
@@ -31,6 +36,14 @@ pub fn try_extract_panic_data(err: &str) -> Option<Vec<Felt252>> {
         }
     }
 
+    if let Some(_captures) = re_entry_point.captures(err) {
+        let panic_data_felts = vec![
+            Felt252::from_bytes_be_slice("ENTRYPOINT_NOT_FOUND".as_bytes()),
+            Felt252::from_bytes_be_slice("ENTRYPOINT_FAILED".as_bytes()),
+        ];
+        return Some(panic_data_felts);
+    }
+
```

Comment:
Could you also explain why we return this specific panic data felts if entry point regex matches? I think we can add brief info about it in the PR description, it might be not clear at first glance.

Reply:
Agreed, maybe we can link some relevant lines from somewhere else?

Reply:
https://github.com/starkware-libs/cairo/blob/2ad7718591a8d2896fec2b435c509ee5a3da9fad/crates/cairo-lang-runner/src/casm_run/mod.rs#L1055-L1057 can be linked in the comment.